uv_magic_uv/muv_transuv_ops.py

```diff
@@ -87,58 +87,75 @@ class MUV_TransUVPaste(bpy.types.Operator):
     bl_options = {'REGISTER', 'UNDO'}
 
     def execute(self, context):
-        props = context.scene.muv_props.transuv
-        active_obj = context.scene.objects.active
-        bm = bmesh.from_edit_mesh(active_obj.data)
+        return MUV_TransUVPasteExecute(self, context, 1);
 
-        if not bm.loops.layers.uv:
-            self.report({'WARNING'}, "No UV Map!!")
-            return {'CANCELLED'}
 
-        uv_layer = bm.loops.layers.uv.verify()  # another approach
+# transfer UV (paste with inverted normals)
+class MUV_TransUVPasteInvertNormals(bpy.types.Operator):
+    """Transfer UV paste invert normals."""
 
-        # get selection history
-        all_sel_faces = [e for e in bm.select_history if isinstance(e, bmesh.types.BMFace) and e.select]
-        if len(all_sel_faces) % 2 != 0:
-            self.report({'WARNING'}, "Number of selected face must be even every part.")
-            return {'CANCELLED'}
+    bl_idname = "uv.muv_transuv_paste_invert_normals"
+    bl_label = "Transfer UV Paste Invert Normals"
+    bl_description = "Transfer UV Paste Invert Normals"
+    bl_options = {'REGISTER', 'UNDO'}
 
-        # parse selection history
-        for i in range(len(all_sel_faces)):
-            if i > 0 and i % 2 != 0:
-                sel_faces = [all_sel_faces[i-1], all_sel_faces[i]]
-                active_face = all_sel_faces[i]
+    def execute(self, context):
+        return MUV_TransUVPasteExecute(self, context, -1);
 
-                # parse all faces according to selection history
-                all_sorted_faces = main_parse(self, active_obj, bm, uv_layer, sel_faces, active_face)
 
-                if all_sorted_faces:
-                    # check amount of copied/pasted faces
-                    if len(all_sorted_faces) != len(props.topology_copied):
-                        self.report({'WARNING'}, "Mesh has different amount of faces!!")
-                        return {'CANCELLED'}
+def MUV_TransUVPasteExecute(self, context, invert_normals):
```

Comment:
Can you rename this function as ```execute_paste()``` ?
Functions are named by snake case (words are separated by ```_```) in this add-on.